src/core/Microsoft.PowerToys.Settings.UI/Views/KeyboardManagerPage.xaml

```diff
@@ -125,15 +146,17 @@
             <Button x:Uid="KeyboardManager_RemapKeyboardButton"
                     Margin="{StaticResource SmallTopMargin}"
                     Style="{StaticResource ButtonRevealStyle}"
-                    Command="{Binding Path=RemapKeyboardCommand}"/>
+                    Command="{Binding Path=RemapKeyboardCommand}"
+                    IsEnabled="{x:Bind Path=ViewModel.Enabled, Mode=OneWay}"/>
 
             <ListView x:Name="RemapKeysList"
                       extensions:ListViewExtensions.AlternateColor="{ThemeResource SystemControlBackgroundListLowBrush}"
-                      ItemsSource="{StaticResource dummyData}"
+                      ItemsSource="{Binding Path=RemapKeys}"
                       ItemTemplate="{StaticResource KeysListViewTemplate}"
                       BorderBrush="{ThemeResource SystemControlForegroundBaseMediumLowBrush}"
                       BorderThickness="1" 
-                      Width="350"
+                      CornerRadius="4"
+                      MinWidth="350"
```

Comment:
I like this cause it solves the issue with larger key names. Is there any quick way we can sync the widths of the two listviews? If it's too complicated we can look into it later.
![image](https://user-images.githubusercontent.com/32061677/79937533-f8fd9b80-840e-11ea-95f0-f320bd083fed.png)

Reply:
I would leave it for later, in any case. I don't really know if we can somehow get the current width of another UI element and update the `ListView`'s width. Maybe from the C# side,